lib/Marlin/Marlin/src/gcode/lcd/M73_PE.cpp

```diff
@@ -78,27 +78,40 @@ oTime2Pause.mInit();
 #if ENABLED(M73_PRUSA)
 void GcodeSuite::M73_PE()
```

Comment:
Missing documentation of this gcode.

Reply:
I miss it too, but there was none. Have no clue what those parameters do.
I just split it in 2, one does parsing and second does actual work